src/cli.rs

```diff
@@ -527,6 +527,157 @@ pub fn build_cli() -> Command {
                         .help("Path to a Docker config.json-like path. Can be used to indicate registry authentication details")
                 )
         )
+        .subcommand(
+            Command::new("bench")
+                .about("Benchmarks a Kubewarden policy from a given URI")
+                .arg(
+                    Arg::new("measurement_time")
+                    .long("measurement-time")
+                    .number_of_values(1)
+                    .value_name("SECONDS")
+                    .help("How long the bench ‘should’ run, num_samples is prioritized so benching will take longer to be able to collect num_samples if the code to be benched is slower than this time limit allowed")
+                )
+                .arg(
+                    Arg::new("num_resamples")
+                    .long("num-resamples")
+                    .number_of_values(1)
+                    .value_name("NUM")
+                    .help("How many resamples should be done")
+                )
+                .arg(
+                    Arg::new("num_samples")
+                    .long("num-samples")
+                    .number_of_values(1)
+                    .value_name("NUM")
+                    .help("Recommended at least 50, above 100 doesn’t seem to yield a significantly different result")
```

Comment:
Even if the name of the argument helps to understand what it does, I think we should add a brief explanation what it means. Something similar of what we do in the `warm_up_time` argument.

Reply:
done